float64[] angles
---
std_msgs/Bool data